hyppo/kgof/tests/test_kernel.py

```diff
@@ -34,13 +35,13 @@ def test_gradX_y(self, n, d):
         k.gradXY_sum(X, Y)
         k.pair_gradX_Y(X, Y)
         k.pair_gradXY_sum(X, Y)
-        k.pair_eval(X, Y)
+        k.pair_eval(X, Y)  # tests usage of multi_compute_kern
```

Comment:
Why are you testing both compute_kern and multi_compute_kern?

Reply:
Was a typo on my part, removing!